Source/CombatExtended/CombatExtended/Projectiles/ArtilleryMarker.cs

```diff
@@ -19,14 +19,32 @@ class ArtilleryMarker : AttachableThing
 
         private int lifetimeTicks = 1800;
 
+        public override string Label
+        {
+            get
+            {
+                return string.Concat("CE_MarkedForArtillery".Translate(), " ", ((int)(lifetimeTicks / 60)).ToString(), " ", "LetterSecond".Translate());
```

Comment:
This `60` should be `GenTicks.TicksPerRealSecond`.  I *doubt* RW will ever change that, but it also annotates the reason for the magic constant.